Assets/RTLTMPro/Scripts/Runtime/LigatureFixer.cs

```diff
@@ -111,15 +111,18 @@ public static void Fix(FastStringBuilder input, FastStringBuilder output, bool f
                         bool isBeforeRTLCharacter = Char32Utils.IsRTLCharacter(nextCharacter);
                         bool isBeforeWhiteSpace = Char32Utils.IsWhiteSpace(nextCharacter);
                         bool isAfterWhiteSpace = Char32Utils.IsWhiteSpace(previousCharacter);
+                        bool isAfterNumber = Char32Utils.IsNumber(previousCharacter, preserveNumbers, farsi);
                         bool isUnderline = characterAtThisIndex == '_';
                         bool isSpecialPunctuation = characterAtThisIndex == '.' ||
                                                     characterAtThisIndex == '،' ||
-                                                    characterAtThisIndex == '؛';
+                                                    characterAtThisIndex == '؛' ||
+                                                    characterAtThisIndex == '؟';
```

Comment:
You can use pattern matching for this, will be much more cleaner.

Reply:
Done